charts/stable/jellyfin/docs/how-to.md

```diff
@@ -0,0 +1,75 @@
+# Jellyfin Setup Guide
+
+Jellyin Media Server is a free fully open-source solution to watch your media from anywhere and our app is a simple way to install it on your TrueNAS SCALE server.
+
+## Requirements
+
+- Jellyfin TrueCharts Chart
+- Installation of the TrueCharts Catalog, starting [here](https://truecharts.org/manual/guides/Adding-TrueCharts)
+
+## Media Storage
+
+- We recommend having configured your media before installing the chart. Most people will have their media inside their TrueNAS box, and for that you can simply follow our [Adding Storage](https://truecharts.org/manual/guides/add-storage/#adding-additional-app-storage) guide to `Add Additional App Storage` to Jellyfin for their media folder.
+- However some will prefer `NFS Shares` for this storage, as they may need to share these folders with other users on other devices outside of TrueNAS, so we recommend following the [using NFS Shares guide](https://truecharts.org/manual/guides/nfs-share).
+
+## Install Jellyfin inside TrueNAS SCALE
+
+- Select `Apps` inside the `TrueNAS` menu, then choose the `Available Applications` tab, and search for jellyfin
+
+![Search Jellyfin](img/SearchJellyfin.png)
+
+- Click the Install button, and you’ll be prompted to set up the Jellyfin software. 
+- Most of the settings can be left at the default values, but ensure you select the correct timezone before scrolling down to the Networking and Services section. Take note of the default port (8096) that Jellyfin is listening on, and move down to Storage and Persistence.
+
+### Jellyfin Configuration Storage
+
+The Jellyfin software itself will have some integrated configuration storage included with a “PVC” storage type - make sure not to change this, as it’s the preferred and supported option from TrueCharts.
+
+![Jellyfin PVC](img/JellyfinPVC.png)
+
+### Jellyfin App Transcode Storage
+
+Users can choose from the default PVC for the Jellyfin App Transcode directory or they can change the App Transcode Storage to the emptyDir type, with the Default storage medium.
+
+More advanced users with a large amount of RAM in their TrueNAS machine might consider changing the transcoding space to use the Memory storage medium. Note that this can potentially consume a large amount of memory if you’re streaming high-definition video content, or multiple streams at the same time.
+
+![Jellyfin App Transcode](img/JellyfinTranscodeDir.png)
+
+### Jellyfin Media Storage
+
+To add your media directory inside Jellyfin you have to click on `Add` in the `Additional App Storage` section
+
+Depedning on your media type (Hostpath or NFS) feel free to follow the guides linked earlier in this guide. As an example we'll add an exisiting `NFS Share` as many users will have that for their media storage.
```

Comment:
Hiya. Just letting you know you have a small spelling error here on Line 42 You have “Depedning” but I think you meant “Depending” still learning GH, but thought I’d let u know. ^.^ Have a good day ^.^